test/hist.jl

```diff
@@ -2,28 +2,28 @@ using StatsBase
 using Base.Test
 
 # test hist
-@test sum(fit(Histogram,[1,2,3]).weights) == 3
-@test fit(Histogram,Int[]).weights == Int[]
-@test fit(Histogram,[1]).weights == [1]
-@test fit(Histogram,[1,2,3],[0,2,4]) == Histogram([0,2,4],[2,1])
-@test fit(Histogram,[1,2,3],[0,2,4]) != Histogram([0,2,4],[1,1])
-@test fit(Histogram,[1,2,3],0:2:4) == Histogram(0:2:4,[2,1])
-@test all(fit(Histogram,[1:100;]/100,0.0:0.01:1.0).weights .==1)
-@test fit(Histogram,[1,1,1,1,1]).weights[1] == 5
-@test sum(fit(Histogram,(rand(100),rand(100))).weights) == 100
+@test sum(fit(Histogram,[1,2,3], closed=:left).weights) == 3  # Explicit arg "closed" during default deprecation
```

Comment:
Looks too verbose to me. Could you simply add a `FIXME: closed` comment at the beginning of the block? A good strategy would be to add a comment at `_check_closed_arg` saying to adapt all places marked `FIXME: closed` when removing that warning, and simply marking the relevant places that way.